### PR TITLE
Feature/tao 7389/delivery executions termination dialog

### PR DESCRIPTION
**Pull request by SashaTAO**
Terminate active delivery executions dialog.

### CODE REVIEW

File: views/js/component/terminateExecutions/terminateExecutions.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+        if (config.csrfToken == undefined) {
+            throw new TypeError('CSRF token is missing');
+        }
+
+        function terminateDeliveryExecutions() {
+            var requestData = {
+                executionsId: config.activeExecutions
+            };
+            requestData[config.csrfToken.name] = config.csrfToken.token;
```
Comment (zagovorichev) [inline views/js/component/terminateExecutions/terminateExecutions.js]: I prefer to use `providers` for this

File: views/js/component/terminateExecutions/terminateExecutions.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                _.forEach(config.data, function (execution) {
+                    config.activeExecutions.push(execution.execution_id);
+
+                    aggregatedData[execution.context_id] = aggregatedData[execution.context_id] || {"total": 0};
+                    aggregatedData[execution.context_id]['label'] = execution.label;
+                    aggregatedData[execution.context_id]['total']++;
+                });
+
+                _.forEach(aggregatedData, function (executionContext) {
+                    config.groupedMessages.push(executionContext.label + " / " + executionContext.total);
```
Comment (zagovorichev) [inline views/js/component/terminateExecutions/terminateExecutions.js]: why do you need this extra forEach? I guess you can do it  in the previous one

File: views/js/component/terminateExecutions/terminateExecutions.js (hunk 1 above)
Comment (zagovorichev) [reply views/js/component/terminateExecutions/terminateExecutions.js]: but you can ask @krampstudio or @jsconan to know if it is required

File: manifest.php [hunk 3]
```diff
@@ -38,8 +38,9 @@
         'taoTestCenter'   => '>=4.6.0',
         'taoResultServer' => '>=7.0.0',
         'taoTaskQueue'    => '>=1.0.0',
+        'taoDelivery'     => '>=12.5.0',
         'taoDeliveryRdf'  => '>=6.0.0',
-        'taoProctoring'   => '>=11.0.0',
+        'taoProctoring'   => '>=12.7.0',
         'taoPublishing'   => '>=1.3.0',
     ),
```
Comment (zagovorichev) [inline manifest.php]: you need to register new services here

File: views/js/component/terminateExecutions/terminateExecutions.js (hunk 1 above)
Comment (SashaTAO) [reply views/js/component/terminateExecutions/terminateExecutions.js]: @jsconan sayid that it's "nice to have" to implement data provider, but in this case it's not really crucial

**Pull request merged**